### PR TITLE
chore(compiler): Format stdlib tests

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

**Review by marcusroberts — APPROVED** [review]
Looks good.    We can format again once I sort the way assert format long lines

File: compiler/test/stdlib/array.test.gr [hunk 1]
```diff
@@ -224,17 +224,18 @@ let arrB = [> 2, 3, 4]
 let arrC = [> 1]
 let arrD = [> 2]
 
-assert Array.product(arrA, arrB) == [>(1,2),(1,3),(1,4),(2,2),(2,3),(2,4)]
-assert Array.product(arrC, arrB) == [>(1,2),(1,3),(1,4)]
-assert Array.product(arrC, arrD) == [>(1,2)]
+assert Array.product(arrA, arrB) ==
+[> (1, 2), (1, 3), (1, 4), (2, 2), (2, 3), (2, 4)]
```
Comment (phated) [inline compiler/test/stdlib/array.test.gr:228]: It's annoying that this doesn't indent. I believe @ospencer pointed that out in #1390

File: compiler/test/stdlib/string.test.gr [hunk 2]
```diff
@@ -77,7 +77,82 @@ assert String.charAt(16, emojis) == ' '
 assert String.charAt(17, emojis) == 't'
 
 // explode tests
-let codes = [> 119, 101, 32, 102, 111, 117, 110, 100, 32, 97, 32, 112, 97, 116, 104, 128640, 32, 116, 111, 32, 103, 114, 101, 97, 116, 110, 101, 115, 115, 127941, 98, 117, 116, 32, 116, 104, 101, 32, 103, 114, 105, 110, 100, 32, 110, 101, 118, 101, 114, 32, 113, 117, 105, 116, 115, 128548, 128170, 127996, 32, 107, 101, 101, 112, 32, 109, 105, 108, 108, 105, 110, 103, 128175, 128293, 128520]
+let codes = [>
```
Comment (phated) [inline compiler/test/stdlib/string.test.gr:80]: @ospencer want to put a `// formatter-ignore` on this?

File: compiler/test/stdlib/string.test.gr [hunk 3]
```diff
@@ -176,71 +257,1135 @@ let processBytes = (b) => {
...
+assert processBytes(String.encode(emojis, String.UTF16_BE)) ==
+[>
```
Comment (phated) [inline compiler/test/stdlib/string.test.gr:311]: Formatter ignore?

File: compiler/test/stdlib/string.test.gr [hunk 4]
```diff
@@ -176,71 +257,1135 @@ let processBytes = (b) => {
...
+  0x3dl,
+  0xddl,
+  0x25l,
+  0xd8l,
+  0x3dl,
+  0xdel,
+  0x08l,
+]
+assert processBytes(String.encode(emojis, String.UTF16_LE)) ==
+[>
```
Comment (phated) [inline compiler/test/stdlib/string.test.gr:478]: Formatter ignore?

File: compiler/test/stdlib/string.test.gr [hunk 5]
```diff
@@ -176,71 +257,1135 @@ let processBytes = (b) => {
...
+assert processBytes(String.encode(emojis, String.UTF32_BE)) ==
+[>
```
Comment (phated) [inline compiler/test/stdlib/string.test.gr:646]: Formatter ignore?

File: compiler/test/stdlib/string.test.gr [hunk 6]
```diff
@@ -176,71 +257,1135 @@ let processBytes = (b) => {
...
+  0x01l,
+  0xf5l,
+  0x25l,
+  0x00l,
+  0x01l,
+  0xf6l,
+  0x08l,
+]
+assert processBytes(String.encode(emojis, String.UTF32_LE)) ==
+[>
```
Comment (phated) [inline compiler/test/stdlib/string.test.gr:945]: Formatter ignore?

File: compiler/test/stdlib/string.test.gr [hunk 7]
```diff
@@ -176,71 +257,1135 @@ let processBytes = (b) => {
...
+  98
+) ==
+emojis
+assert String.decodeRange(
+  String.encodeAt(emojis, String.UTF16_LE, Bytes.make(500), 51),
+  String.UTF16_LE,
+  51,
+  164
+) ==
+emojis
```
Comment (phated) [inline compiler/test/stdlib/string.test.gr:1280]: These are rough 🙁

**Review by phated — COMMENTED** [review]
This looks good. I think I just want some formatter ignore comments on the large character arrays in string tests.

**Comment by marcusroberts** [comment]
@ospencer  can you redo with #1400  it should solve all these

**Review by phated — APPROVED** [review]
Approved! Thanks for doing this. Can't believe I forgot when adding the formatter CI.

I re-ran the only failure. Good to go after that.

**Pull request merged**